telegram-translation-shortcuts.user.js

```diff
@@ -66,20 +67,78 @@ function scrollItems (down) {
   }
 }
 
+/**
+ * A div (key-add-suggestion-wrap) contains the input-form to Add translation with Submit and Cancel buttons
+ * This function will check if that div is collapsed or not. If it is collapsed,
+ * it's className will change to 'key-add-suggestion-wrap collapsed'
+ *
+ * @function inputWrapCollapsed() -- Checks if element by class 'key-add-suggestion-wrap' is exactly the same. If not the same, then it is collapsed
+ * @returns True if collapsed
+ * @returns False if NOT collapsed
+ */
+function inputWrapCollapsed () {
+  if (document.getElementsByClassName('key-add-suggestion-wrap')[0].className === 'key-add-suggestion-wrap') { // checking if it is NOT collapsed
```

Comment:
Should I change the className to `inputWrapClosed`, to make it easier to understand the overall code?

Reply:
`inputWrapOpen` ? (won't be understood by those who haven't seen the html of the website)

Reply:
the working of the function will be changed accordingly!

Reply:
Since it returns a boolean, it’s good practice to prefix it with ‘is’/‘has’ (so it’s immediately clear what it does) – `isTranslationInputOpen()` sounds most natural to me working with that rule.